emails/templates/emails/first_time_user.html

```diff
@@ -2,8 +2,10 @@
   Note that Django only loads strings from some Fluent files.
   See privaterelay/ftl_bundles.py.
 {% endcomment %}
+{% load cache %}
```

Comment:
TIL [Template Fragment Caching](https://docs.djangoproject.com/en/4.2/topics/cache/#template-fragment-caching) 😎

Reply:
This is how I learned it too!